syft/serde.py

```diff
@@ -69,7 +69,13 @@
 
 
 # High Level Public Functions (these are the ones you use)
-def serialize(obj: object, simplified=False) -> bin:
+def serialize(
+    obj: object,
+    simplified=False,
+    force_no_compression=False,
```

Comment:
Pleas add explanation for what these parameters mean (also what to do when they conflict)